region/crateria/east/The Moat.json

```diff
@@ -408,6 +429,27 @@
         "A beam shot, Missile, or Super can be used."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Come in Shinecharging, Leave With Temporary Blue",
+      "entranceCondition": {
```

Comment:
With enough speed you can jump over all the water, and shoot to open the door before morphing.
Or spacejump across

Reply:
Also added strats for jumping across the Moat from left to right, which has more significant speed requirements. Added a new property `minTiles` to `comeInShinecharging` to be able to express this.